get-parity.sh

```diff
@@ -30,9 +30,9 @@ check_os() {
 
 get_package() {
 	if [ "$RELEASE" = "beta" ]; then
-		LOOKUP_URL="$VANITY_SERVICE_URL&os=$PKG&version=beta-release"
+		LOOKUP_URL="$VANITY_SERVICE_URL&os=$PKG&version=$VERSION_BETA"
```

Comment:
I think this needs a `v` prefix. e.g.:

:x: https://vanity-service.parity.io/parity-binaries?architecture=x86_64&format=markdown&os=linux&version=2.0.3

:heavy_check_mark: https://vanity-service.parity.io/parity-binaries?architecture=x86_64&format=markdown&os=linux&version=v2.0.3